src/System.Net.WebHeaderCollection/pkg/System.Net.WebHeaderCollection.pkgproj

```diff
@@ -8,9 +8,7 @@
     <ProjectReference Include="..\ref\System.Net.WebHeaderCollection.csproj">
       <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Net.WebHeaderCollection.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-    </ProjectReference>
+    <ProjectReference Include="..\src\System.Net.WebHeaderCollection.builds" />
```

Comment:
This package never had a RID before.  The one configuration of the library was not Windows specific.  It was an authoring error in the original source.